hedera-node/hedera-network-admin-service-impl/src/main/java/com/hedera/node/app/service/networkadmin/impl/handlers/NetworkTransactionGetReceiptHandler.java

```diff
@@ -34,6 +34,7 @@
 /**
  * This class contains all workflow-related functionality regarding {@link HederaFunctionality#TRANSACTION_GET_RECEIPT}.
  */
+// use RecordCache for this
```

Comment:
Should this be a todo?

Reply:
it will be fixed in the coming pr we are working on I can remove it or it will be removed as part of the next prs

Reply:
That would make it `@todo('PR#')`  (makes it easier to ensure it's not forgotten).